frontend/test/playwright/e2e/sources.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from "@playwright/test"
+
+import { getH1 } from "~~/test/playwright/utils/components"
+import { preparePageForTests } from "~~/test/playwright/utils/navigation"
+
+test.describe.configure({ mode: "parallel" })
+
+test("sources table has links to source pages", async ({ page }) => {
+  await preparePageForTests(page, "xl", {
+    features: { additional_search_views: "on" },
+  })
+  await page.goto("/sources")
+  await page.getByRole("link", { name: "Animal Diversity Web" }).click()
+  await page.waitForURL("/source/animal-diversity-web")
```

Comment:
The test is failing, but I think this would fix it :thinking: 

```suggestion
  await page.waitForURL("/images/source/animal-diversity-web")
```

Reply:
I ended up replacing Animal Diversity Web with Flickr, so that I don't need to add any more tapes.